src/Input/ToggleSwitch/ToggleSwitch.story.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+
+import ToggleSwitch from './ToggleSwitch';
+import { ToggleSwitchProps } from './ToggleSwitch.interface';
+import StorybookComponent from '../../../stories/StorybookComponent';
+import { SecondaryColor } from '../..';
+
+const onClick = (e: any) => {}
+
+const props: ToggleSwitchProps = {
+  defaultActive: false,
+  activeBallColor: SecondaryColor.white,
+  inactiveBallColor: SecondaryColor.white,
+  activeBackgroundColor: SecondaryColor.actionblue,
+  inactiveBackgroundColor: SecondaryColor.grey,
+  activeIconName: 'eye',
```

Comment:
Since `activeIconName` and `inactiveIconName` are closely related, consider putting them together as an object instead. E.g.:

```
iconOptions: {
    active: 'eye',
    inactive: 'eye-slashed',
  },
```